Mejoras en el diseño de la página de contratos

YanmirG opened the pull request:
Se han eliminado estilos de fondo en `contratos.css` y se ha ajustado la altura mínima de la cabecera. En `Index.cshtml`, se han añadido variables en `ViewBag` para el título y subtítulo, eliminando la sección de cabecera anterior. En `_Layout.cshtml`, se ha implementado la carga condicional de CSS y scripts específicos para el controlador "Contratos".

YanmirG committed:
Mejoras en el diseño de la página de contratos

Se han eliminado estilos de fondo en `contratos.css` y se ha ajustado la altura mínima de la cabecera.
En `Index.cshtml`, se han añadido variables en `ViewBag` para el título y subtítulo, eliminando la sección de cabecera anterior.
En `_Layout.cshtml`, se ha implementado la carga condicional de CSS y scripts específicos para el controlador "Contratos".

Jonathan03R commented on capa_presentacion/Views/Shared/_Layout.cshtml:
```diff
@@ -26,40 +30,39 @@
+    }
+
     <script>
         (function () {
```
La lógica no debe estar en el mismo HTML , *separa responsabilidades

YanmirG committed:
Añadidos DTOs y mejoras en la gestión de datos

Se han añadido múltiples clases DTO en `ReferenciasDTO.cs`, incluyendo `TipoSalarioDTO` y `TipoJornadaDTO`, con documentación XML.

En `Contratos.cs`, se han actualizado los métodos para obtener datos de la base de datos, reemplazando procedimientos almacenados por consultas SQL directas y añadiendo manejo de excepciones.

Se ha reestructurado `contratos.js` para mejorar la gestión de eventos y la apertura/cierre de modales, además de implementar un manejo de errores más claro.

En `Index.cshtml`, se han modificado los botones para abrir modales, eliminando `onclick` y utilizando atributos `data-action`. También se ha mejorado la inclusión del script `contratos.js` en la sección `Scripts`.

Se ha eliminado la inclusión condicional del script en `_Layout.cshtml` y se ha corregido la ruta del script en `capa_presentacion.csproj`.

Jonathan03R commented:
Mala estructura en los DTO
